### ISSUE TITLE
Error when creating new Message Template directly with Custom Sound

### ISSUE THREAD

**Issue by Hayato2846** [PhenomDevel/PhenomRaidTools#25]
**Describe the bug**
When you create a new Message Template (Type Advanced) an lua error occurs while checking Custom Sound. You can still edit and manage that Message Template including Custom Sound after a /reload.

**To Reproduce**
1. Create a new Message Template via 'New'
2. Change the Template Name, e.g. 'Test - 123'
3. Change Message Type to 'Advanced'
4. Check Option 'Custom Sound'
5. Lua Error + Message Template Name resets to generic one again, e.g. 'template35027':

```
1x PhenomRaidTools\Source/GUI/Message.lua:506: attempt to index local 'action' (a nil value)
[string "@PhenomRaidTools\Source/GUI/Message.lua"]:506: in function <PhenomRaidTools\Source/GUI/Message.lua:489>
[string "@PhenomRaidTools\Source/GUI/Message.lua"]:564: in function `MessageWidget'
[string "@PhenomRaidTools\Source/GUI/Template.lua"]:75: in function <...rfacePhenomRaidTools\Source/GUI/Template.lua:72>
[string "=[C]"]: ?
[string "@Ace3\AceGUI-3.0\AceGUI-3.0-41.lua"]:72: in function <Ace3\AceGUI-3.0\AceGUI-3.0.lua:70>
[string "@Ace3\AceGUI-3.0\AceGUI-3.0-41.lua"]:306: in function `Fire'
[string "@Ace3\AceGUI-3.0-41\widgets\AceGUIContainer-TabGroup.lua"]:156: in function `SelectTab'
[string "@PhenomRaidTools\Source/GUI/AceHelper.lua"]:166: in function `ReSelectTab'
[string "@PhenomRaidTools\Source/GUI/Message.lua"]:381: in function <PhenomRaidTools\Source/GUI/Message.lua:378>
[string "=[C]"]: ?
[string "@Ace3\AceGUI-3.0\AceGUI-3.0-41.lua"]:72: in function <Ace3\AceGUI-3.0\AceGUI-3.0.lua:70>
[string "@Ace3\AceGUI-3.0\AceGUI-3.0-41.lua"]:306: in function `Fire'
[string "@Ace3\AceGUI-3.0-41\widgets\AceGUIWidget-CheckBox.lua"]:68: in function <...ns\Ace3\AceGUI-3.0\widgets\AceGUIWidget-CheckBox.lua:57>
```

**Expected behavior**
No lua error occurs.

**Comment by PhenomDevel** [comment]
Thanks for the report.

This problem should be solved in 2.2.7.4.

Please update to the current version and let me know if this fixed the issue for you.

**Comment by Hayato2846** [comment]
Problem solved.

Thanks!
